.ci/test.sh

```diff
@@ -50,15 +50,21 @@ if [[ $TRAVIS == "true" ]] && [[ $TASK == "check-docs" ]]; then
     exit 0
 fi
 
-if [[ $TASK == "lint" ]]; then
+if [[ $TRAVIS == "true" ]] && [[ $TASK == "lint" ]]; then
     conda install -q -y -n $CONDA_ENV \
         pycodestyle \
-        pydocstyle \
-        r-lintr
+        pydocstyle
     pip install --user cpplint
     echo "Linting Python code"
     pycodestyle --ignore=E501,W503 --exclude=./compute,./.nuget . || exit -1
     pydocstyle --convention=numpy --add-ignore=D105 --match-dir="^(?!^compute|test|example).*" --match="(?!^test_|setup).*\.py" . || exit -1
+    # stringi needs to be installed separate from r-lintr
+    # to avoid issues like 'unable to load shared object stringi.so'
+    conda install -q -y -n $CONDA_ENV \
+        r-stringi
```

Comment:
I think it could be 
```
conda install -q -y -n $CONDA_ENV \
    pycodestyle \
    pydocstyle \
    r-stringi  # stringi needs to be installed separate from r-lintr to avoid issues like 'unable to load shared object stringi.so'
conda install -q -y -n $CONDA_ENV \
    -c conda-forge \
        r-lintr>=2.0
```
to avoid multiple `conda install ...` commands and keep them together in one place for a better readability.

Reply:
sure I'm ok with that

Reply:
done